module/observations/helper.js

```diff
@@ -1006,6 +1006,14 @@ module.exports = class ObservationsHelper {
                 name : requestedData.name,
                 description : requestedData.description
               };
+
+              if( requestedData.taskId ) {
+                  solutionInformation["taskId"] = solutionInformation["taskId"];
```

Comment:
@aman-tunerlabs - Not required inside solutionInformation as already present outside.